tests/test_integration.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+
+@pytest.fixture
+def firefox_options(firefox_options):
+    """Set Firefox Options."""
+    firefox_options.headless = True
+    return firefox_options
+
+@pytest.mark.nondestructive
+def test_html_loads_correctly(base_url, selenium):
+    selenium.get(base_url)
+    header = selenium.find_element_by_css_selector(".content div > h1")
+    assert "Kinto Administration" in header.text
```

Comment:
Here I would just leave `"Administration" in header.text` because once the server infos have been fetched we replace Kinto by the instance name (eg. `Kinto DEV`, `Remote Settings STAGE`, ...). This way this script can be used on different instances

Reply:
The string is generated from the server configuration (eg. `{project_name} Administration`).
Testing that it ends with `Administration` should be enough.

You can try your script with https://kinto.dev.mozaws.net/v1/admin/  or https://settings-writer.prod.mozaws.net/v1/admin/ in case you have VPN access